tests/Fixtures/Integration/priority/single_space_after_construct,braces.test

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Integration of fixers: single_space_after_construct,braces.
```

Comment:
(yes, I know we have open discussion here. let's discuss in other thread to not block this PR)